eng/AzureTestLab.ps1

```diff
@@ -250,3 +250,55 @@ function Invoke-TestRunnerFromTeamCity($testType) {
 
     Copy-Item $zipPath -Destination artifacts -FromSession $session -Recurse
 }
+
+# Run performance tests analysis in the Azure Test Lab, assuming the following environment variables, write test results to an artifacts folder.
+#
+# Relies on environment variables:
+#   $env:AzureTestVmPassword
+#   $env:AzureTestVmUsername
+function Invoke-PerfTestReportFromTeamCity($testType) {
+    if (!(Test-Path artifacts)) { New-Item -ItemType Directory -Force -Path artifacts | Out-Null }
+
+    $securePassword = $env:AzureTestVmPassword | ConvertTo-SecureString -AsPlainText -Force
+    $credential = New-Object -TypeName System.Management.Automation.PSCredential -ArgumentList $env:AzureTestVmUsername, $securePassword
+
+    $sessionOptions = New-PSSessionOption -SkipCACheck -SkipCNCheck -SkipRevocationCheck
+    $session = New-PSSession -UseSSL -Port 5986 -ComputerName 'edfi-perf-test.southcentralus.cloudapp.azure.com' -Credential $credential -SessionOption $sessionOptions -ConfigurationName SecondHopConfiguration
+
+    # Set $testRunnerPath locally and remotely.
+    $testRunnerPathR = Invoke-Command -Session $session {
+        $testRunnerPathR = Get-Content "C:\Users\edFiAdmin\deployed-test-runner-path.txt" -Raw
+        $testRunnerPathR
+    }
+
+    # Set $testResultsPath locally and remotely.
+    if ($testType -eq "volume") {
+        $testResultsPathR = Invoke-Command -Session $session {
+            $testResultsPathR = "C:\Octopus\Applications\Perf-v3\Suite-3-Performance-Testing\1.1.0-pre-0251\ReportV"
```

Comment:
Do not hardcode the path, it changes for each new deployment - See the reference to C:\Users\edFiAdmin\deployed-test-runner-path.txt above. That contains where the tests are deployed.